ServiceBus/ServiceBus/ProductService.svc.cs

```diff
@@ -159,30 +156,21 @@ public SharedLibs.DataContracts.Product EditProduct(Guid guid, string name, doub
 
                     if (!String.IsNullOrWhiteSpace(name) && name.Length <= 50 && name != editableProduct.Name)

```

Comment:
I completely forgot about checking length of the name, wow. Incredible. But anyway, I was wondering are 50 characters enough for a name of a product? Just a rhetorical question, nothing necessary.